src/libraries/System.IO.Packaging/src/System/IO/Packaging/Package.cs

```diff
@@ -401,32 +401,63 @@ public PackagePartCollection GetParts()
 
                 PackUriHelper.ValidatedPartUri partUri;
 
+                var uriComparer = Comparer<PackUriHelper.ValidatedPartUri>.Default;
```

Comment:
Note above on LOC 393 there should be 2 more doc'd exceptions (FileFormat and InvalidOperation) based on https://github.com/dotnet/runtime/pull/108677/commits/b4ccde403bdb8976d3ad4768d16002fa232f1ac0#diff-e0565b90c6887f4c5e77f09ebf8ae412b72414d0d4a9c49fe3bdf7b82525b81cR383-R384